.changeset/twelve-turtles-cry.md

```diff
@@ -0,0 +1,8 @@
+---
+"@tiptap/extension-bubble-menu": patch
+"@tiptap/react": patch
+"@tiptap/vue-2": patch
+"@tiptap/vue-3": patch
+---
+
+Fix the bug where the bubble menu does not hide
```

Comment:
I think a more precise description would help here (don't specify the bug but explain more what changed as this will be generated into the changelog and helps people to understand what's new. This wouldn't tell a user looking at the changelog nothing really).

Reply:
```suggestion
Added a `focusout` event listener to handle blur events on the bubble menu element, ensuring the menu closes properly when blurred.
```